lib/core-integration/src/Test/Integration/Scenario/API/Byron/Wallets.hs

```diff
@@ -358,6 +385,113 @@ spec = describe "BYRON_WALLETS" $ do
             , expectField (#name . #getApiT . #getWalletName) (`shouldBe` updatedName)
             ]
 
+    it "BYRON_COIN_SELECTION_00 - \
+        \No coin selection on Byron random" $ \ctx -> do
+        rnW <- emptyRandomWallet ctx
+        shW <- emptyWallet ctx
+        (addr:_) <- fmap (view #id) <$> listAddresses @n ctx shW
+        let payments = NE.fromList [ AddressAmount addr (Quantity minUTxOValue) ]
+        selectCoins @_ @'Byron ctx rnW payments >>= flip verify
+            [ expectResponseCode @IO HTTP.status403
+            , expectErrorMessage errMsg403NotAnIcarusWallet
+            ]
+
+    it "BYRON_COIN_SELECTION_01 - \
+        \A singleton payment is included in the coin selection output." $
+        \ctx -> do
+            source <- fixtureIcarusWallet ctx
+            target <- emptyWallet ctx
+            targetAddress : _ <- fmap (view #id) <$> listAddresses @n ctx target
+            let amt = Quantity minUTxOValue
+            let payment = AddressAmount targetAddress amt
+            let output = ApiCoinSelectionOutput targetAddress amt
+            let isValidDerivationPath path =
+                    ( length path == 5 )
+                    &&
+                    ( [ ApiT $ DerivationIndex $ getIndex purposeBIP44
+                      , ApiT $ DerivationIndex $ getIndex coinTypeAda
+                      , ApiT $ DerivationIndex $ getIndex @'Hardened minBound
+                      ] `isPrefixOf` NE.toList path
+                    )
+            selectCoins @_ @'Byron ctx source (payment :| []) >>= flip verify
+                [ expectResponseCode HTTP.status200
+                , expectField #inputs
+                    (`shouldSatisfy` (not . null))
+                , expectField #inputs
+                    (`shouldSatisfy` all
+                        (isValidDerivationPath . view #derivationPath))
+                , expectField #change
+                    (`shouldSatisfy` all
+                        (isValidDerivationPath . view #derivationPath))
```

Comment:
Hi @piotr-iohk, just wondering if should we expect to find at least one change value here?

So something like:

```hs
, expectField #change
    (`shouldSatisfy` (not . null))
```